From 0fa2d7f968c90adebae14fcf3dac14962e80fd79 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Mon, 7 Oct 2013 14:08:25 -0700
Subject: [PATCH 32/39] Ticket #53 - Need to update supported locales
 Cleaning up typos and format.

https://fedorahosted.org/389/ticket/53

Reviewed by rmeggins (Thank you, Rich!!)
(cherry picked from commit 4a1dac27fbc0b46b3a68c19658ba41731b80c575)
(cherry picked from commit 4b105d06e99aa8a0adb902a34cc843795569301a)
---
 ldap/schema/slapd-collations.conf |   80 ++++++++++++++++++------------------
 1 files changed, 40 insertions(+), 40 deletions(-)

diff --git a/ldap/schema/slapd-collations.conf b/ldap/schema/slapd-collations.conf
index 55fe893..e8d05fe 100644
--- a/ldap/schema/slapd-collations.conf
+++ b/ldap/schema/slapd-collations.conf
@@ -119,8 +119,8 @@ collation az "" "" 1 3	2.16.840.1.113730.3.3.2.73.1	az
 collation az Latn "" 1 3	2.16.840.1.113730.3.3.2.74.1	az-Latn
 collation az Latn_AZ "" 1 3	2.16.840.1.113730.3.3.2.75.1	az-Latn-AZ
 collation bn "" "" 1 3	2.16.840.1.113730.3.3.2.76.1	bn
-collation bn BD "" 1 3	2.16.840.1.113730.3.3.2.77.1	bn	bn-BD
-collation bn IN "" 1 3	2.16.840.1.113730.3.3.2.78.1	bn	bn-IN
+collation bn BD "" 1 3	2.16.840.1.113730.3.3.2.77.1	bn-BD
+collation bn IN "" 1 3	2.16.840.1.113730.3.3.2.78.1	bn-IN
 collation bs "" "" 1 3	2.16.840.1.113730.3.3.2.79.1	bs	bs-BA
 collation chr "" "" 1 3	2.16.840.1.113730.3.3.2.80.1	chr
 collation chr US "" 1 3	2.16.840.1.113730.3.3.2.81.1	chr-US
@@ -136,7 +136,7 @@ collation en BE "" 1 3	2.16.840.1.113730.3.3.2.90.1	en-BE
 collation en BW "" 1 3	2.16.840.1.113730.3.3.2.91.1	en-BW
 collation en BZ "" 1 3	2.16.840.1.113730.3.3.2.92.1	en-BZ
 collation en GU "" 1 3	2.16.840.1.113730.3.3.2.93.1	en-GU
-collation en HK "" 1 3	2.16.840.1.113730.3.3.2.94.1	en-HK
+collation en GY "" 1 3	2.16.840.1.113730.3.3.2.94.1	en-GY
 collation en HK "" 1 3	2.16.840.1.113730.3.3.2.95.1	en-HK
 collation en IN "" 1 3	2.16.840.1.113730.3.3.2.96.1	en-IN
 collation en JM "" 1 3	2.16.840.1.113730.3.3.2.97.1	en-JM
@@ -180,46 +180,46 @@ collation es VE "" 1 3	2.16.840.1.113730.3.3.2.134.1	es-VE
 collation fa "" "" 1 3	2.16.840.1.113730.3.3.2.135.1	fa	fa-IR
 collation fil "" "" 1 3	2.16.840.1.113730.3.3.2.136.1	fil	fil-PH
 collation fo "" "" 1 3	2.16.840.1.113730.3.3.2.137.1	fo	fo-FO
-collation fr BF "" 1 3	2.16.840.1.113730.3.3.2.138.1	fr	fr-BF
-collation fr BI "" 1 3	2.16.840.1.113730.3.3.2.139.1	fr	fr-BI
-collation fr BJ "" 1 3	2.16.840.1.113730.3.3.2.140.1	fr	fr-BJ
-collation fr BL "" 1 3	2.16.840.1.113730.3.3.2.141.1	fr	fr-BL
-collation fr CD "" 1 3	2.16.840.1.113730.3.3.2.142.1	fr	fr-CD
-collation fr CF "" 1 3	2.16.840.1.113730.3.3.2.143.1	fr	fr-CF
-collation fr CG "" 1 3	2.16.840.1.113730.3.3.2.144.1	fr	fr-CG
-collation fr CI "" 1 3	2.16.840.1.113730.3.3.2.145.1	fr	fr-CI
-collation fr CM "" 1 3	2.16.840.1.113730.3.3.2.146.1	fr	fr-CM
-collation fr DJ "" 1 3	2.16.840.1.113730.3.3.2.147.1	fr	fr-DJ
-collation fr GA "" 1 3	2.16.840.1.113730.3.3.2.148.1	fr	fr-GA
-collation fr GN "" 1 3	2.16.840.1.113730.3.3.2.149.1	fr	fr-GN
-collation fr GP "" 1 3	2.16.840.1.113730.3.3.2.150.1	fr	fr-GP
-collation fr GQ "" 1 3	2.16.840.1.113730.3.3.2.151.1	fr	fr-GQ
-collation fr KM "" 1 3	2.16.840.1.113730.3.3.2.152.1	fr	fr-KM
-collation fr LU "" 1 3	2.16.840.1.113730.3.3.2.153.1	fr	fr-LU
-collation fr MC "" 1 3	2.16.840.1.113730.3.3.2.154.1	fr	fr-MC
-collation fr MF "" 1 3	2.16.840.1.113730.3.3.2.155.1	fr	fr-MF
-collation fr MG "" 1 3	2.16.840.1.113730.3.3.2.156.1	fr	fr-MG
-collation fr ML "" 1 3	2.16.840.1.113730.3.3.2.157.1	fr	fr-ML
-collation fr MQ "" 1 3	2.16.840.1.113730.3.3.2.158.1	fr	fr-MQ
-collation fr NE "" 1 3	2.16.840.1.113730.3.3.2.159.1	fr	fr-NE
-collation fr RE "" 1 3	2.16.840.1.113730.3.3.2.160.1	fr	fr-RE
-collation fr RW "" 1 3	2.16.840.1.113730.3.3.2.161.1	fr	fr-RW
-collation fr SN "" 1 3	2.16.840.1.113730.3.3.2.162.1	fr	fr-SN
-collation fr TD "" 1 3	2.16.840.1.113730.3.3.2.163.1	fr	fr-TD
-collation fr TG "" 1 3	2.16.840.1.113730.3.3.2.164.1	fr	fr-TG
+collation fr BF "" 1 3	2.16.840.1.113730.3.3.2.138.1	fr-BF
+collation fr BI "" 1 3	2.16.840.1.113730.3.3.2.139.1	fr-BI
+collation fr BJ "" 1 3	2.16.840.1.113730.3.3.2.140.1	fr-BJ
+collation fr BL "" 1 3	2.16.840.1.113730.3.3.2.141.1	fr-BL
+collation fr CD "" 1 3	2.16.840.1.113730.3.3.2.142.1	fr-CD
+collation fr CF "" 1 3	2.16.840.1.113730.3.3.2.143.1	fr-CF
+collation fr CG "" 1 3	2.16.840.1.113730.3.3.2.144.1	fr-CG
+collation fr CI "" 1 3	2.16.840.1.113730.3.3.2.145.1	fr-CI
+collation fr CM "" 1 3	2.16.840.1.113730.3.3.2.146.1	fr-CM
+collation fr DJ "" 1 3	2.16.840.1.113730.3.3.2.147.1	fr-DJ
+collation fr GA "" 1 3	2.16.840.1.113730.3.3.2.148.1	fr-GA
+collation fr GN "" 1 3	2.16.840.1.113730.3.3.2.149.1	fr-GN
+collation fr GP "" 1 3	2.16.840.1.113730.3.3.2.150.1	fr-GP
+collation fr GQ "" 1 3	2.16.840.1.113730.3.3.2.151.1	fr-GQ
+collation fr KM "" 1 3	2.16.840.1.113730.3.3.2.152.1	fr-KM
+collation fr LU "" 1 3	2.16.840.1.113730.3.3.2.153.1	fr-LU
+collation fr MC "" 1 3	2.16.840.1.113730.3.3.2.154.1	fr-MC
+collation fr MF "" 1 3	2.16.840.1.113730.3.3.2.155.1	fr-MF
+collation fr MG "" 1 3	2.16.840.1.113730.3.3.2.156.1	fr-MG
+collation fr ML "" 1 3	2.16.840.1.113730.3.3.2.157.1	fr-ML
+collation fr MQ "" 1 3	2.16.840.1.113730.3.3.2.158.1	fr-MQ
+collation fr NE "" 1 3	2.16.840.1.113730.3.3.2.159.1	fr-NE
+collation fr RE "" 1 3	2.16.840.1.113730.3.3.2.160.1	fr-RE
+collation fr RW "" 1 3	2.16.840.1.113730.3.3.2.161.1	fr-RW
+collation fr SN "" 1 3	2.16.840.1.113730.3.3.2.162.1	fr-SN
+collation fr TD "" 1 3	2.16.840.1.113730.3.3.2.163.1	fr-TD
+collation fr TG "" 1 3	2.16.840.1.113730.3.3.2.164.1	fr-TG
 collation ga "" "" 1 3	2.16.840.1.113730.3.3.2.165.1	ga
 collation ga IE "" 1 3	2.16.840.1.113730.3.3.2.166.1	ga-IE
 collation ga IN "" 1 3	2.16.840.1.113730.3.3.2.167.1	ga-IN
-collation ha Latn "" 1 3	2.16.840.1.113730.3.3.2.168.1	ha-Latn
-collation ha Latn GH 1 3	2.16.840.1.113730.3.3.2.169.1	ha-Latn-GH
-collation ha Latn NE 1 3	2.16.840.1.113730.3.3.2.170.1	ha-Latn-NE
-collation ha Latn NG 1 3	2.16.840.1.113730.3.3.2.171.1	ha-Latn-NG
-collation he "" "" 1 3	2.16.840.1.113730.3.3.2.172.1	he	he-IL
-collation hi "" "" 1 3	2.16.840.1.113730.3.3.2.173.1	hi	hi-IN
-collation hy "" "" 1 3	2.16.840.1.113730.3.3.2.174.1	hy	hy-AM
-collation id "" "" 1 3	2.16.840.1.113730.3.3.2.175.1	id
-collation id ID "" 1 3	2.16.840.1.113730.3.3.2.176.1	id-ID
-collation ig "" "" 1 3	2.16.840.1.113730.3.3.2.177.1	ig	ig-NG
+collation ha "" "" 1 3	2.16.840.1.113730.3.3.2.168.1	ha
+collation ha Latn "" 1 3	2.16.840.1.113730.3.3.2.169.1	ha-Latn
+collation ha Latn GH 1 3	2.16.840.1.113730.3.3.2.170.1	ha-Latn-GH
+collation ha Latn NE 1 3	2.16.840.1.113730.3.3.2.171.1	ha-Latn-NE
+collation ha Latn NG 1 3	2.16.840.1.113730.3.3.2.172.1	ha-Latn-NG
+collation he "" "" 1 3	2.16.840.1.113730.3.3.2.173.1	he	he-IL
+collation hi "" "" 1 3	2.16.840.1.113730.3.3.2.174.1	hi	hi-IN
+collation hy "" "" 1 3	2.16.840.1.113730.3.3.2.175.1	hy	hy-AM
+collation id "" "" 1 3	2.16.840.1.113730.3.3.2.176.1	id-ID
+collation id ID "" 1 3	2.16.840.1.113730.3.3.2.177.1	ig-NG
 collation it IT "" 1 3	2.16.840.1.113730.3.3.2.178.1	it-IT
 collation ka "" "" 1 3	2.16.840.1.113730.3.3.2.179.1	ka
 collation ka GE "" 1 3	2.16.840.1.113730.3.3.2.180.1	ka-GE
-- 
1.7.1

